simulation/chart.py

```diff
@@ -1,14 +1,134 @@
 # Copyright 2022-, Semiotic AI, Inc.
 # SPDX-License-Identifier: Apache-2.0
 
+from enum import Enum
 from subprocess import Popen
-from typing import Dict, Optional, Tuple
+from typing import Any, Dict, NamedTuple, Optional, Tuple, TypedDict
 
 import ffmpeg
 import pyqtgraph as pg
 from pyqtgraph.Qt import QtCore, QtGui
 
 
+class RGBAColor(NamedTuple):
+    """An RGB color value represents RED, GREEN, and BLUE light sources
+    with alpha channel- which specifies the opacity for a color."""
+
+    R: int
+    """Red [0-255]"""
+    G: int
+    """Green [0-255]"""
+    B: int
+    """Blue [0-255]"""
+    A: Optional[int]
+    """Alpha [0-255]"""
+
+
+class NamedColor(Enum):
```

Comment:
I think that's overkill.
Try this instead: https://peps.python.org/pep-0586/

Reply:
https://github.com/semiotic-ai/autoagora-agents/pull/21/commits/22d207415aed5a8c972ca517ba0ea026431270ee